### ISSUE TITLE
Support auto-prefixing of CSS in inline styles in HTML templates

### ISSUE THREAD

**Issue by mdittmer** [foam-framework/foam#404]
Currently, HTML templates that contain <tag style="styles"> do not run styles through the CSS parser/generator. We should make it so. A temporary fix for existing code is checked in at 25decb0a26c8d533b81e80c2ed386f15025fedad.

**Comment by shepheb** [comment]
I believe the CSS parser has since fixed this. @mdittmer to close if that's true.

**Comment by mdittmer** [comment]
Not true. What's meant by "styles" is inline `<foo style="bar:baz">`. The CSS parser is not wired up to DOM elements to manage prefixing in this case.
